src/protocolgame.cpp

```diff
@@ -248,6 +248,10 @@ void ProtocolGame::login(uint32_t characterId, uint32_t accountId, OperatingSyst
 			eventConnect = g_scheduler.addEvent(
 			    createSchedulerTask(1000, [=, thisPtr = getThis(), playerID = foundPlayer->getID()]() {
 				    thisPtr->connect(playerID, operatingSystem);
+				    Player* player = g_game.getPlayerByID(playerID);
```

Comment:
probably to remove and change to use `if (thisPtr->player)` instead

Reply:
It seems correct to me; the player may no longer exist after 1 second. The same valid. is present in `connect`.